server/routes.js

```diff
@@ -1,18 +1,24 @@
-var bodyParser = require('body-parser');
+var bodyParser = require("body-parser");
+import webhooks from "/server/slack/webhooks.js";
```

Comment:
eslint problems? ping @distalx

Reply:
@sebbel  It's coming from my [Prettier](https://prettier.io/) config. Will revert and disable for this project.